packages/amplify-cli/src/extensions/amplify-helpers/searchable-push-checks.ts

```diff
@@ -0,0 +1,16 @@
+import { EnvironmentDoesNotExistError, exitOnNextTick, stateManager, $TSAny, $TSContext } from 'amplify-cli-core';
+import { ResourceConstants } from 'graphql-transformer-common';
+import { printer } from 'amplify-prompts';
+import { readJsonFile } from './read-json-file';
+
+
+export function searchablePushChecks(context: $TSContext): void {
+  const currEnv = context.amplify.getEnvInfo().envName;
+  const teamProviderInfoFilePath = context.amplify.pathManager.getProviderInfoFilePath();
+  const teamProviderInfo = readJsonFile(teamProviderInfoFilePath);
```

Comment:
nit: can we use stateManager.getTeamProviderInfo

Reply:
Fixed